src/components/Network.tsx

```diff
@@ -60,34 +56,40 @@ const useStyles = makeStyles((theme) => ({
 export const NetworkSides = () => {
   const classes = useStyles();
   const { sourceChainDetails, targetChainDetails } = useSourceTarget();
+  const dbSource = useDashboard(ChainDetails.SOURCE);
+  const dbTarget = useDashboard(ChainDetails.TARGET);
+
   return (
     <Box marginY={2} className={classes.main}>
-      <Box p={1}>{sourceChainDetails.sourceChain}</Box>
+      <Box p={1} className={classes.statsEntry}>
+        <Typography variant="h4">{sourceChainDetails.sourceChain}</Typography>
```

Comment:
I think it worth to sync this branch with master since we have put back the links to polkadotJS UI:
https://github.com/paritytech/parity-bridges-ui/blob/master/src/components/Network.tsx#L66